translator/sample files/cotrip_data/cotrip_2.json

```diff
@@ -0,0 +1,36 @@
+{

```

Comment:
Is this file used?

Reply:
we used that file when we run the program